FreydCategoriesForCAP/gap/FreydCategory.gi

```diff
@@ -1344,7 +1344,7 @@ InstallGlobalFunction( INSTALL_FUNCTIONS_FOR_FREYD_CATEGORY,
                 sink := [ mor1, mor2 ];
                 
                 # compute the universal morphism
-                uni := UniversalMorphismFromDirectSum( underlying_category, diagram, sink );
+                uni := UniversalMorphismFromDirectSum( underlying_category, diagram, range, sink );
```

Comment:
If the 3rd argument of `UniversalMorphismFromDirectSum` now is the range of the result, then this line is correct.

Is this version of `UniversalMorphismFromDirectSum` documented?

Reply:
> If the 3rd argument of `UniversalMorphismFromDirectSum` now is the range of the result, then this line is correct.

It is: it's the test object, i.e. common range of the morphisms in `sink`, and thus the range of the universal morphism.

> Is this version of `UniversalMorphismFromDirectSum` documented?

Yes, it is. In case you looked for a four argument version and could not find it: CAP operations are not documented with the category as the first argument, so in this case one has to look for the three argument version.